src/types/venues.ts

```diff
@@ -143,6 +144,7 @@ export interface Venue {
   termsAndConditions: TermOfService[];
   showRadio?: boolean;
   showBadges?: boolean;
+  showNametags?: boolean;
```

Comment:
Still seems to be a duplicate here with line 73

Reply:
Thanks for the comments and suggestions. 
This duplicate seems to be the pattern used here, where variables are declared for the Venue and Venue_v2_AdvancedConfig separately.

Reply:
Ah, cool, that makes sense (I was only looking at the diff changes specifically, hadn't fully expanded the surrounding code)

This should be fine as is then :)